libs/docs/src/navigation/no-version.yaml

```diff
@@ -19,38 +19,16 @@ top:
         href: ./what-is-a-devfile
       - title: Benefits of devfile
         href: ./benefits-of-devfile
-      - title: Users of devfile
-        href: ./users-of-devfile
-      - title: How to work with devfiles
-        href: ./how-to-work-with-devfiles
+      - title: Devfile ecosystem
+        href: ./devfile-ecosystem
+  - title: Get started
+    links:
       - title: Developing with devfiles
         href: ./developing-with-devfiles
-  - title: Quick start
-    links:
-      - title: Overview
-        href: ./overview
-      - title: Versions
-        href: ./versions
-      - title: Metadata
-        href: ./metadata
-      - title: Library
-        href: ./library
-      - title: Resources
-        href: ./resources
       - title: Integrate with editors
         href: ./integrate-with-editors
-  - title: Get started
-    links:
-      - title: Devfile ecosystem
-        href: ./devfile-ecosystem
-      - title: Application developer
-        href: ./application-developer
-      - title: Enterprise architect and runtime provider
-        href: ./enterprise-architect-and-runtime-provider
-      - title: Registry administrator
-        href: ./registry-administrator
-      - title: Technology and tool builders
-        href: ./technology-and-tool-builders
```

Comment:
> Devfile perspective pages Application developer, Enterprise architect and runtime provider, and Registry administrator and Technology and tool builders should be under the General rather than Getting Started. They should be group together as persona docs that are probably referred to at the end of the *Devfile ecosystem*

These documents should be at the end of Devfile ecosystem, at the moment they are not.

Reply:
I was planning on removing the persona docs and instead have that content in the landing page, but I'll add them back here for now and we can consider it in a different issue